tests/triggers.spec.ts

```diff
@@ -259,3 +259,24 @@ test('clicking #shadowInInvoker should show then hide popover', async ({
   await page.click('#shadowInvokedHost');
   await expect(popover).toBeHidden();
 });
+
+test('clicking button[popovertarget=defaultPopover] should not trigger popover when defaultPrevented is true', async ({
+  page,
+}) => {
+  const popover = (await page.locator('#defaultPopover')).nth(0);
+  await expect(popover).toBeHidden();
+  await page.evaluate(() => {
+    const button = document.querySelector(
+      'button[popovertarget="defaultPopover"]',
+    );
+    button?.addEventListener(
+      'click',
+      (event) => {
+        event.preventDefault();
+      },
+      { capture: true },
```

Comment:
Is capture needed here? There shouldn't be any click event listeners on children of the button, and it looks like it works without. If it is helpful to the test, can you add a comment?

Reply:
Yeah definitely not necessary. 😅